docs/reference/settings/security-settings.asciidoc

```diff
@@ -2609,7 +2609,7 @@ setting, this would be `transport.profiles.$PROFILE.xpack.security.ssl.key`.
 
 [discrete]
 [[ip-filtering-settings]]
-==== IP filtering settings
+=== IP filtering settings
```

Comment:
I think the existing indentation level is incorrect for the right sidebar. This section should be at the same level as TLS settings.